data/data_generator/data_generator.h

```diff
@@ -57,10 +57,22 @@ class DataGenerator {
       }
     }
 
+    // Scramble the raw information
+    int8_t scramble_buffer[LdpcEncodingInputBufSize(lc.BaseGraph(),
+                                                    lc.ExpansionFactor())];
+    for (size_t i = 0;
+         i < LdpcEncodingInputBufSize(lc.BaseGraph(), lc.ExpansionFactor());
+         i++) {
+      scramble_buffer[i] = information.at(i);
+    }
+    WlanScramble(scramble_buffer,
+                 LdpcEncodingInputBufSize(lc.BaseGraph(), lc.ExpansionFactor()),
+                 kScramblerInitState);
+
```

Comment:
Might be a good idea to add the scrambler to the configuration values.  Default to true, but add the hooks to disable.

Reply:
Yes, let me work on that.